satellite_populate/commands.py

```diff
@@ -23,28 +23,55 @@
 
 Use :code:`$ satellite-populate --help` for more info
 """
-import sys
-
 import click
+import os
+import sys
+import yaml
 
 from satellite_populate.constants import TEST_DATA
 from satellite_populate.main import populate
 
+CONFIG_FILE = '.satellite_populate.yaml'
+SATELLITE_POPULATE_FILE = 'SATELLITE_POPULATE_FILE'
+
+
+def _read_populate_settings(config_file):
+    """Parse satellite-populate configuration"""
+    if config_file.endswith(('.yml', '.yaml', 'json')):
+        with open(config_file) as file:
+            settings = yaml.load(file)
+            if type(settings) == dict:
+                return settings
+    return {}
+
+
+def configure():
+    """Read satellite-populate settings file."""
+    config = os.path.join(os.environ[SATELLITE_POPULATE_FILE], CONFIG_FILE) \
+        if SATELLITE_POPULATE_FILE in os.environ.keys() \
+        else os.path.join(os.environ['HOME'], CONFIG_FILE)
```

Comment:
This would read better if it wasn't in the single line if format. Can you break it up into the traditional style?